### DISCUSSION TITLE
Env.Sus. Glossary/Encyclopedia

### DISCUSSION THREAD

**Discussion by mkorbi** [cncf/tag-env-sustainability#15] [General]
The idea would be to create a glossary or even a small kind of encyclopedia for Environmental Sustainability wordings.
This can start with simple things like what is Green House Gas (GHG), terms like PUE - what do they mean, and how do they measure. 

We would need to first check if there is something specific like this or something more generic. We shouldn't re-invent the wheel, but it maybe would make sense to build a reference base we can link to.

For some/many terms, this could be also just further links, especially if there are ongoing unclear discussions.

Overall, this would be a never-ending activity and everyone can contribute to this easily.

**Comment by wojtekidd** [comment]
[Let's start here](https://www.wikiwand.com/en/Glossary_of_climate_change) - grab all the basic ones and then add all the cloud and tech-related WDYT?

**Reply by mkorbi** [reply]
this is beautiful! 
maybe we can get in contact with the folks there and leave them some back links for "Read more here". Awesome collection.

**Reply by mkorbi** [reply]
Just saw it is under CC BY-SA 4.0, think we should have the same ambitions. Guess we need to clarify this shortly with CNCF but if we add a [CC](https://creativecommons.org/licenses/by-sa/4.0/) license to the glossary it should do the thing.

**Comment by wojtekidd** [comment]
[Climate Change Vocabulary ](https://www.health.state.mn.us/communities/environment/climate/docs/film/vocab_list.pdf)
[Key Terms You Need to Know to Understand Climate Change](https://www.climaterealityproject.org/blog/key-terms-you-need-understand-climate-change)
[Glossary of Climate Change Terms](https://19january2017snapshot.epa.gov/climatechange/glossary-climate-change-terms_.html)

The question is - do we link or do we use these to create our own glossary 🤔

**Reply by mkorbi** [reply]
I would leave them as the input, maybe as the source where possible, but also to underline that we have somewhere in our heads the Cloud Native context. Best of both worlds.

**Comment by catblade** [comment]
Request came out of the other meeting this week to wait on the glossary until we have enough knowledge of what words are being used and their definitions from currently defined projects.  Defining words when there may be nuances to their usages within documents may limit us and cause confusion.

**Reply by mkorbi** [reply]
I don't see that issue. Many things are a moving target, from a glossary, to a list of suiting foundations or a landscape.
Nothing is written in stone.

**Reply by catblade** [reply]
I am asking we wait until we have more information from current landscapes on definitions.  We are just beginning here.  Starting a glossary before we've properly researched the landscape feels like an ordering issue.

**Reply by mkorbi** [reply]
The market is way farther ahead than we are. General definitions are existing and available, and general terms and methods like PUE (Power Usage Effectiveness) or SIEC (Server Idle Energy Coefficiency) maybe need a home in our context. If there are projects who defined new words or methods that can be easily added.

**Reply by leonardpahlke** [reply]
I also don't see a problem with starting now. It might actually help us to make sure that we all have the same understanding at the beginning. I assume that the glossary will be developed more open-ended, so that we can push other projects in parallel from which we adapt technical terms in the glossary.

**Reply by all-green-dev** [reply]
I agree with @mkorbi. I don't think this is an ordering issue, since both tasks are likely to be an ongoing undertaking and seeing, which words the different members of the WG use and/or think are important, could help us discuss our focus and be a perfect starting ground to exchange and discuss ideas for the future. Realizing which parts, words or ideas we see outside of the scope of this working group is vital, because I think that no idea that helps improve environmental sustainability should be left unexplored. Therefore, people who are interested in these topics can immediately seek a new home and breading ground for their ideas, maybe at one of the organizations from our friends list.

**Comment by mkorbi** [comment]
I have created an initial working document for the ground work.

https://docs.google.com/document/d/1-xIXsLASbapbcGuk-_XJhK6GEyujoWOCzqtjhJWJ_fE/edit

**Comment by mkorbi** [comment]
Closing that discussion, PR is opened for the initial glossary
